benchmarks/benchmarkSuite/dsl/parameter_study.json

```diff
@@ -0,0 +1,304 @@
+{
+  "cases": [
+    {
+      "template_case": "/home/henning/libsAndApps/FoamAdapter/benchmarks/benchmarkSuite/dsl/templates/3DCube",
```

Comment:
I guess this is a generated file? I guess it shouldn't be in here otherwise the hardcoded path is an issue.